src/makernote_int.cpp

```diff
@@ -81,6 +81,10 @@ std::string getExiv2ConfigPath() {
 }
 
 std::string readExiv2Config(const std::string& section, const std::string& value, const std::string& def) {
+  // Silence unused parameter warning when EXV_ENABLE_INIH is undefined.
+  (void)section;
+  (void)value;
```

Comment:
Maybe move into #else below?

Reply:
done

Reply:
Another option would be to use `[[maybe_unused]]` directly in the function signature, what would end up in less LoCs.

Reply:
Thanks, that's cleaner. I've updated it.